subsys/bluetooth/controller/hci/hci.c

```diff
@@ -2310,7 +2310,12 @@ int hci_acl_handle(struct net_buf *buf, struct net_buf **evt)
 		return -ENOBUFS;
 	}
 
+#if defined(CONFIG_BT_LL_SW_SPLIT)
 	pdu_data = (void *)node_tx->pdu;
+#else
+	pdu_data = (void *)((struct radio_pdu_node_rx *)node_tx)->pdu_data;
```

Comment:
This is node_tx... there is not change in this struct between old and new arch implementation

Reply:
Reverted change